app/services/to_collection_form/mapper.rb

```diff
@@ -27,8 +28,26 @@ def params
         description: CocinaSupport.abstract_for(cocina_object:), # Cocina abstract maps to Collection description
         contact_emails_attributes: CocinaSupport.contact_emails_for(cocina_object:),
         related_links_attributes: CocinaSupport.related_links_for(cocina_object:),
+        managers_attributes:,
+        depositors_attributes:,
         version: cocina_object.version
       }
     end
+
+    def managers_attributes
+      return [] if collection&.managers.blank?
```

Comment:
Is this protection necessary? Is collection ever nil? Could this just be `return [] if collection.nil?`?

Reply:
Actually, I don't think so. This appears to be another case where the collection being nil was only a test artifact and easily fixed there.